vm/systemSmartContracts/eei.go

```diff
@@ -120,9 +120,11 @@ func (host *vmContext) GetContract(address []byte) (vm.SystemSmartContract, erro
 
 // GetStorageFromAddress gets the storage from address and key
 func (host *vmContext) GetStorageFromAddress(address []byte, key []byte) []byte {
+	//log.Warn("get storage", "addr", address, "key", key)
```

Comment:
commented code here, on L127 and on L157

Reply:
done